src/containers/BooksList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+function BooksList(props) {
+  const { books } = props;
+  const bookList = books.map(book => (<Book book={book} key={Math.random()} />));
```

Comment:
- [ ] [OPTIONAL] The key of a map component is serious business! Don't use Math.random() for that.
The key should be an immutable and unique value. You can set it as the name of the book or add an id attribute to the book object when it's created based on the number of books in the store.

[Read more on keys and lists here.](https://reactjs.org/docs/lists-and-keys.html#keys)